src/components/NewLocationPage/SparkLineBlock/utils.ts

```diff
@@ -34,6 +36,21 @@ export const SparkLineToExploreMetric: {
   [SparkLineMetric.RATIO_BEDS_WITH_COVID]: ExploreMetric.RATIO_BEDS_WITH_COVID,
 };
 
+// TODO (chelsi) - fix need for all metrics as keys
+export const MetricToSparkLine: {
+  [metric in Metric]: SparkLineMetric;
+} = {
+  [Metric.WEEKLY_CASES_PER_100K]: SparkLineMetric.WEEKLY_CASES_PER_100K,
+  [Metric.ADMISSIONS_PER_100K]: SparkLineMetric.ADMISSIONS_PER_100K,
+  [Metric.RATIO_BEDS_WITH_COVID]: SparkLineMetric.RATIO_BEDS_WITH_COVID,
+  // Not used, just included for the sake of typescript:
+  [Metric.CASE_DENSITY]: SparkLineMetric.RATIO_BEDS_WITH_COVID,
+  [Metric.CASE_GROWTH_RATE]: SparkLineMetric.RATIO_BEDS_WITH_COVID,
+  [Metric.HOSPITAL_USAGE]: SparkLineMetric.RATIO_BEDS_WITH_COVID,
+  [Metric.POSITIVE_TESTS]: SparkLineMetric.RATIO_BEDS_WITH_COVID,
+  [Metric.VACCINATIONS]: SparkLineMetric.RATIO_BEDS_WITH_COVID,
+};
```

Comment:
Do we need to use `[metric in Metric]`? Perhaps we can write this as a function that takes an instance of a metric and returns a `SparkLineMetric`? If not, we can leave as is since I see you put in a TODO in here anyway.

Reply:
yeah gonna keep as is for now